espanol/comida/comida.md

```diff
@@ -0,0 +1,4 @@
+Aguantose las náuseas, y tras enjugarse el sudor, pensó que nuevas le traería el capitán de las tropas:
+
+[El ejercito había vencido al enemigo](victoria/victoria.md)
+[Una orda de ogros había atravesado la frontera norte](ogros/ogros.md)
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.